### ISSUE TITLE
Crash after splash (logo) screen

### ISSUE THREAD

**Issue by andy5995** [arcayn/bos-wars#2]
When I run the program, I get the splash screen with the Boswars logo, but when I do a single left-click, the window disappears. The console reports:

```
libpng error: IDAT: invalid distance too far back
Error reading the PNG file.
Can't load the graphic `graphics/ui/ui_minimap.png'
```

When I tried copying the png files from the boswars svn repo, the crash doesn't happen.

```
andy@oceanus:~/src/bos-wars$ diff -q graphics/ui ../boswars-svn/graphics/ui
Common subdirectories: graphics/ui/cursors and ../boswars-svn/graphics/ui/cursors
Common subdirectories: graphics/ui/patcheditor and ../boswars-svn/graphics/ui/patcheditor
Files graphics/ui/ui_bpanel_200x144.png and ../boswars-svn/graphics/ui/ui_bpanel_200x144.png differ
Files graphics/ui/ui_bpanel_200x264.png and ../boswars-svn/graphics/ui/ui_bpanel_200x264.png differ
Files graphics/ui/ui_bpanel_200x432.png and ../boswars-svn/graphics/ui/ui_bpanel_200x432.png differ
Files graphics/ui/ui_bpanel_200x624.png and ../boswars-svn/graphics/ui/ui_bpanel_200x624.png differ
Files graphics/ui/ui_bpanel_200x864.png and ../boswars-svn/graphics/ui/ui_bpanel_200x864.png differ
Files graphics/ui/ui_info.png and ../boswars-svn/graphics/ui/ui_info.png differ
Files graphics/ui/ui_minimap.png and ../boswars-svn/graphics/ui/ui_minimap.png differ```

**Comment by arcayn** [comment]
Interesting, I cannot replicate this issue. I will take a closer look at the differences between the files but I imagine it's to do with differing versions of libpng.

I believe it would be a good idea to switch out the images regardless, I'll need to have a look at them with my version of libpng.

**Comment by andy5995** [comment]
I think this was fixed with the recent release of Boswars 2.8 

https://codeberg.org/boswars/boswars/src/branch/master/CHANGELOG
